src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/FieldMaker.java

```diff
@@ -94,6 +95,9 @@ private Field makeFieldImpl(SQLExpr expr, String alias, String tableAlias) throw
                 return makeFilterMethodField(mExpr, alias);
             }
 
+            if ((SQLFunctions.builtInFunctions.contains(methodName.toLowerCase())) && Strings.isNullOrEmpty(alias)) {
```

Comment:
I wonder is the first check necessary? `(SQLFunctions.builtInFunctions.contains(methodName.toLowerCase()))`. Any edge case?

Reply:
Yep, it was necessary. Some tests were broken, for example:
`
SELECT address, CASE WHEN gender='0' THEN 'aaa' ELSE 'bbb' END AS a2345, count(age) FROM accounts GROUP BY terms('field'='address','execution_hint'='global_ordinals'), a2345
`
For the part of "terms", what we expected was `"terms(field=address,execution_hint=global_ordinals)"`, what we actually got was `"terms(com.alibaba.druid.sql.ast.expr.SQLBinaryOpExpr@fedcc4,com.alibaba.druid.sql.ast.expr.SQLBinaryOpExpr@f90c3f62)"`

Reply:
I see. Thanks!